lib/presentation/widget/upload_file/upload_file_widget.dart

```diff
@@ -57,7 +62,8 @@ class _UploadFileWidgetState extends State<UploadFileWidget> {
 
   @override
   Widget build(BuildContext context) {
-    final UploadFileArguments arguments = ModalRoute.of(context).settings.arguments;
+    final UploadFileArguments arguments =
+        ModalRoute.of(context).settings.arguments;
```

Comment:
:(

Reply:
I used auto format code '_' This line exceed max code lengh :(